web/modules/custom/collection_page/config/install/node.type.collection_page.yml

```diff
@@ -0,0 +1,28 @@
+langcode: en
+
+config_devel:
+  - node.type.collection_page
+  - field.storage.user.og_group_ref
+  - field.storage.node.body
+  - field.field.node.collection_page.body
+  - field.field.node.collection_page.og_group_ref
+  - core.base_field_override.node.collection_page.promote
+  - core.base_field_override.node.collection_page.status
+  - core.entity_form_display.node.collection_page.default
+  - core.entity_view_display.node.collection_page.default
+  - core.entity_view_display.node.collection_page.teaser
```

Comment:
This is interesting, I haven't seen these exports of config_devel data in individual config items before. I wonder what this is used for.